packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -108,14 +108,23 @@ export const userCanEditUsersBannedUserIds = (currentUser: DbUser|null, targetUs
   )
 }
 
-const postHasModerationGuidelines = (post: PostsBase | DbPost) => {
+const postHasModerationGuidelines = (
+  post: PostsBase|PostsModerationGuidelines|DbPost,
+): boolean => {
   // Because of a bug in Vulcan that doesn't adequately deal with nested fields
   // in document validation, we check for originalContents instead of html here,
   // which causes some problems with empty strings, but should overall be fine
-  return ('moderationGuidelines' in post && post.moderationGuidelines?.originalContents) || post.moderationStyle
+  return !!(
```

Comment:
I'm a little uncertain about the changes to this function. I think it's correct, but I'm not 100%.